packages/ra-ui-materialui/src/field/TranslatableFieldsTabContent.tsx

```diff
@@ -109,3 +119,31 @@ const Root = styled('div', {
     border: `1px solid ${theme.palette.divider}`,
     borderTop: 0,
 }));
+
+const getRecordForLocale = (record: RaRecord, locale: string) => {
+    // Get all paths of the record
+    const paths = getRecordPaths(record);
+
+    // For each path, if a path ends with the locale, set the value of the path without the locale
+    // to the value of the path with the locale
+    const recordForLocale = paths.reduce((acc, path) => {
+        if (path.includes(locale)) {
```

Comment:
This is too weak IMO, as it will match 'francois' for the 'fr' locale. I'd use a RegExp, testing for entries like 'XXX.[locale]' only.

Reply:
`path` is an array, not a string